transform/convert_quadkey_and_Vertical_id.go

```diff
@@ -672,7 +988,32 @@ func convertVerticallIDToBit(vZoom int64, vIndex int64, outputZoom int64, maxHei
 
 }
 
-func convertZToAltitudekey(inputIndex int64, inputZoom int64, outputZoom int64, zBaseExponent int64, zBaseOffset int64) ([]int64, error) {
+// ConvertZToAltitudekey (拡張)空間IDのz成分をaltitudekeyに高度変換する。
+//
+// 変換前と変換後の精度差によって出力されるaltitudekeyの個数は増減する。
+//
+// 引数 :
+//
+//	inputIndex : 変換対象の(拡張)空間IDのz成分(fインデックス)
+//
+//	inputZoom : 変換対象の(拡張)空間IDのズームレベル(zインデックス)
+//
+//	outputZoom : 変換後のaltitudekeyの精度指定
+//
+//	zBaseExponent : 変換後のaltitudekeyの高さが1mとなるズームレベル
+//
+//	zBaseOffset : ズームレベルがzBaseExponentのとき、高度0mにおけるaltitudekeyのインデックス値
+//
+// 戻り値 :
+//
+//	変換後のaltitudekeyのスライス
+//
+// 戻り値(エラー) :
+//
+//	以下の条件に当てはまる場合、エラーインスタンスが返却される。
+//	 入力インデックス不正       ：inputIndexにそのズームレベル(inputZoom)で存在しないインデックス値が入力されていた場合。
+//	 出力インデックス不正       ：出力altitudekeyが出力ズームレベル(outputZoom)で存在しないインデックス値になった場合。
+func ConvertZToAltitudekey(inputIndex int64, inputZoom int64, outputZoom int64, zBaseExponent int64, zBaseOffset int64) ([]int64, error) {
```

Comment:
Rename it to `ConvertZToMinMaxAltitudekey`.
And, implement it as same as `ConvertAltitudeKeyToMinMaxZ`.

Reply:
It was remained since renaming existing function `ConvertZToMinMaxAltitudekey` may break backward compatibility.
is this PR to major version change?

Reply:
refactored in [9dbe76d](https://github.com/trajectoryjp/spatial_id_go/pull/29/commits/9dbe76d0364339543beb838c362307d85579d5e6)

note: this PR is minor change